istioctl/pkg/install/verify.go

```diff
@@ -138,13 +138,17 @@ func NewVerifyCommand() *cobra.Command {
 		fileNameFlags = &genericclioptions.FileNameFlags{
 			Filenames: &filenames,
 			Recursive: boolPtr(true),
+			Usage:     "Istio YAML installation file.",
 		}
 	)
 	verifyInstallCmd := &cobra.Command{
 		Use:   "verify-install",
 		Short: "Verifies Istio Installation Status",
 		Long: `
-		verify-install Verifies Istio Installation Status
+		verify-install Verifies Istio Installation Status against the installation file
+		you specified when you installed Istio. It loops through all the installation
+		resources defined in your installation files and reports whether all of them are
+		in ready status. It will report failure when any of them are not ready.
```

Comment:
could you also submit a PR on using this cmd at the end of the install doc?   

https://istio.io/docs/setup/kubernetes/helm-install/
https://istio.io/docs/setup/kubernetes/quick-start/

Reply:
also does this cmd work with the new CRDs which are in istio-init now?

Reply:
yes

Reply:
cool!